less/c2corg_ui.less

```diff
@@ -508,8 +508,22 @@ app-simple-search {
   border-radius: 6px;
   padding-left: 10px;
   margin-right: 10px;
-  background: @C2C-orange;
+  background: rgba(255,255,255,0.25);
 
+  .form-control {
+        box-shadow: none;
+    }
```

Comment:
All those lines are not correctly indented (indentation = 2 cols + selectors of the same level should be aligned)